editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -358,14 +399,20 @@ const RowForBaseControl = React.memo((props: RowForBaseControlProps) => {
         variant='<--1fr--><--1fr-->|-18px-|'
       >
         {propertyLabel}
-        <div ref={dataPickerButtonData.setReferenceElement}>
+        <div
+          style={{
+            minWidth: 0, // this ensures that the div can never expand the allocated grid space
```

Comment:
Nice fix. 
off topic - at the future I think we should actually fix it in the `UIGridRow` component, as I imagine it will be a common issue (since the use of `fr` can yield unexpected results)

Reply:
good point!